includes/ucf-post-list-config.php

```diff
@@ -462,14 +462,15 @@ public static function delete_configurable_options() {
 		 * @since 1.0.0
```

Comment:
Can we add a description of the `$layout` param to this comment block?  (`@param string $layout Name of the layout requested for the shortcode`)